kingfisher_scrapy/spiders/moldova.py

```
@@ -41,7 +41,8 @@ def parse_list(self, response):
         base_url = 'http://public.eprocurement.systems/ocds/tenders/'
         data = response.json()
         # The last page returns an empty JSON object.
-        if not data:
+        # Occasional error response with HTTP 200 code.
+        if not data or ('name' in data and data['name'] == 'Error'):
```

When returning an Error, we should return a file error, similar to: https://github.com/open-contracting/kingfisher-collect/blob/4265ea3808a0475ccbda9428d95fac35d799f020/kingfisher_scrapy/spiders/chile_base.py#L78-L80